tests/unit/linalg/test_operator.cpp

```diff
@@ -31,23 +31,19 @@ TEST_CASE("Operator", "[Operator]")
    {
       // When cg is in a product (on the right), we require cg->iterative_mode to be false.
 
-      ProductOperator *product = NULL;
```

Comment:
@hughcars looks like a bug from your merge commit with `master`?

Reply:
No, this fixes a warning/error from the compiler about unused variables from merge then rebuilding. To test the throws `new` isn't needed, nor a variable definition. This issue is in and fixed within loads of PRs right now, as it blocks the CI after I presume some flags got upgraded somewhat recently.

Reply:
Maybe bug is the wrong word, I guess what I am saying is this change from `master` does not belong in this PR.